### PR TITLE
Remove articles, fix 'About' footer link

### PR DESCRIPTION
**Pull request by BrewCityBoy**
**Taken from issue #284: JIRA 5240: Remove the blog from github.cfpb.io 

_Remove the blog

~~Remove https://cfpb.github.io/articles/~~
~~Remove all https://cfpb.github.io/articles/ child pages~~
Remove/Hide link to cfpb.github.io/articles
Remove the "Learn more" to the blog from https://cfpb.github.io/_

**To test:**
Visit http://localhost:4000/ and verify "Learn more" link has been removed.

Hover over "About" page in footer in footer and verify URL change (/cfpb-open-tech)

- Once at the "About" page, hover over the "CFPB Open Tech" logo/text in upper left corner and verify URL is set back to the landing page. (cfpb.github.io)

~~Navigate to cfpb.github.io/articles and verify page has been removed.~~
cfpb.github.io/articles will still be live, but without linkage on any page that will take the user to 'articles'.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 exclude: [CONTRIBUTING.md, Gemfile, Gemfile.lock, LICENSE, README.md, TERMS.md]
-permalink: /articles/:title/
+# permalink: /articles/:title/
```
Comment (anselmbradford) [inline _config.yml:2]: Do we want to delete this entirely vs commenting it out? Is there some reason to leave it here commented out?

Comment (anselmbradford) [reply _config.yml]: That is my understanding

File: _includes/footer.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
                 </ul>
                 <ul class="footer-links_secondary cf">
                     <li>
-                        <a href="http://cfpb.github.io/articles/introducing-cfpb-open-tech/">About</a>
+                        <a href="http://cfpb.github.io/cfpb-open-tech/">About</a>
```
Comment (anselmbradford) [inline _includes/footer.html:14]: Would it work to make this a relative URL, or is that going to mess it up on GitHub pages?

**Review by anselmbradford — CHANGES_REQUESTED** [review]
Follow up from 7/6/23 meeting:

- Add posts back, but don't link to them.
- Change about link in footer to relative link.
- Change cfpb-lighthouse URL on homepage to cfgov-lighthouse.

File: _includes/footer.html (hunk 2 above)
Comment (BrewCityBoy) [reply _includes/footer.html:14]: It should not be an issue, using a relative url. That update has been applied.

File: _config.yml (hunk 1 above)
Comment (BrewCityBoy) [reply _config.yml]: Leaving this line in, as the articles are to remain live until further notice.

**Comment by BrewCityBoy** [comment]
Per request (from Adam), I added a section for the Design System in featured projects, on the landing page.

File: _includes/footer.html (hunk 2 above)
Comment (anselmbradford) [reply _includes/footer.html:14]: Did the relative URL not get pushed?

File: index.html [hunk 3]
```diff
@@ -69,6 +69,18 @@ <h3 class="featured-project_head">
                 <a class="call-to-action" href="http://cfpb.github.io/api/ccdb/">API docs</a>
             </div>
         </li>
+        <li class="featured-project">
+            <div class="featured-project_img__ds"></div>
+            <h3 class="featured-project_head">
+                <a href="http://cfpb.github.io/design-manual/">Design System</a>
```
Comment (anselmbradford) [inline index.html:75]: This is linking to the design-manual (which redirects, but then we get a banner). 
It should be https://cfpb.github.io/design-system/

**Review by anselmbradford — CHANGES_REQUESTED** [review]
Font is still not Avenir in the heading because it is set to `"Avenir Next W01", Arial, sans-serif;` in the intro and hero styles, which is not the current designator to pick up Avenir. We want to set it to Avenir in the body (as you have done) and remove the places it is separately set.

**Comment by anselmbradford** [comment]
Layout seems not so great at desktop. We probably want to keep the four-up layout across tablet and desktop, or adjust the sizing so it's one row at desktop. Sounds like something you and Sam should work out in another PR after this

Screen Shot 2023-07-11 at 12 55 40 PM

**Comment by BrewCityBoy** [comment]
> Layout seems not so great at desktop. We probably want to keep the four-up layout across tablet and desktop, or adjust the sizing so it's one row at desktop. Sounds like something you and Sam should work out in another PR after this
> 
> Screen Shot 2023-07-11 at 12 55 40 PM

Agreed. I actually changed the margin and padding because I didn't think it looked so hot. I was thinking about changing it to two per row, but I thought it would be out of scope. I just wanted to get the DS on the landing page as requested and we can follow it up with styling and customization.

**Review by anselmbradford — APPROVED** [review]

**Pull request merged**